src/main/features/response/views/reject-all-of-claim.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t("Money claim") %}
+
+{% set heading = t("I reject all of the claim") %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        <div class="form-group {% if error %} form-group-error {% endif %}">
```

Comment:
That is not needed as `radioGroup` macro does it for us. Please check what `radioGroup` is doing and remove unnecessary markup.